spec/unit/harvestdor/indexer/metrics_spec.rb

```diff
@@ -1,45 +1,45 @@
 require 'spec_helper'
 
 describe Harvestdor::Indexer::Metrics do
-  it "should record successes" do
+  it 'records successes' do
     expect { subject.success! }.to change { subject.success_count }.from(0).to(1)
   end
 
-  it "should record errors" do
+  it 'records errors' do
     expect { subject.error! }.to change { subject.error_count }.from(0).to(1)
   end
 
-  describe "#total" do
-    it "should be the sum of the successes and errors" do
-      expect do 
+  describe '#total' do
+    it 'is the sum of the successes and errors' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.